compose/gradle.properties

```
@@ -0,0 +1,18 @@
+#
+# Copyright 2025 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+artifactRedirection.targetNames=android
+artifactRedirection.groupIdReplacement=org.jetbrains.->androidx.
```

Syntax looks weird here, but anyway, it's better to be more explicit here
```suggestion
artifactRedirection.groupIdReplacement=org.jetbrains.compose.->androidx.compose.
```

---

Also changed for other modules, and remove the dot.

Rechecked the resulted module files